java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -1207,4 +1207,21 @@ CompletableFuture<String> restore(
      * }</pre>
      */
     CompletableFuture<Long> sortStore(String key, String destination);
+
+    /**
+     * Blocks the current client until all the previous write commands are successfully transferred
+     * and acknowledged by at least <code>numreplicas</code> of replicas. If <code>timeout</code> is
+     * reached, the command returns even if the specified number of replicas were not yet reached.
+     *
+     * @param numreplicas The number of replicas to reach.
+     * @param timeout The timeout value specified in milliseconds.
```

Comment:
Tell that 0 is no timeout (and in transaction too)

Reply:
0 is actually for blocking forever -- unless we treat that as the same thing as no timeout?

Reply:
Oh I messed things. Tell what 0 timeout means.